### PR TITLE
Document disableWorkerMessageHandler

### PR DESCRIPTION
**Pull request by jakearchibald**
We're going to use this in SVGOMG, so I figured it should be documented 😄

### CODE REVIEW

**Comment by RunDevelopment** [comment]
I didn't even know we had this option. Thank you for documenting it!

Unfortunately, you documented the wrong file. `prism.js` is generated. Please document `components/prism-core.js` instead, run `npm ci && npm run build`, and commit all changes.

**Comment by jakearchibald** [comment]
Done!

File: components/prism-core.js [hunk 1]
```diff
@@ -49,6 +49,27 @@ var Prism = (function (_self) {
 		 * @public
 		 */
 		manual: _self.Prism && _self.Prism.manual,
+		/**
+		 * By default, if Prism is in a web worker, it assumes its in a worker it created itself, so it uses
```
Comment (RunDevelopment) [inline components/prism-core.js:53]:
```suggestion
		 * By default, if Prism is in a web worker, it assumes that it is in a worker it created itself, so it uses
```

**Review by RunDevelopment — CHANGES_REQUESTED** [review]
Sorry for the delay @jakearchibald!

**Comment by jakearchibald** [comment]
Updated

File: components/prism-core.js [hunk 2]
```diff
@@ -49,6 +49,27 @@ var Prism = (function (_self) {
 		 * @public
 		 */
 		manual: _self.Prism && _self.Prism.manual,
+		/**
+		 * By default, if Prism is in a web worker, it assumes that it is in a worker it created itself, so it uses
+		 * `addEventListener` to communicate with its parent instance. However, if you're using Prism manually in your
+		 * own worker, you don't want it to do this.
+		 *
```
Comment (RunDevelopment) [inline components/prism-core.js:56]: Our linter really doesn't like trailing spaces.
```suggestion
		 *
```

**Comment by jakearchibald** [comment]
Fixed

**Review by RunDevelopment — APPROVED** [review]

**Comment by RunDevelopment** [comment]
Thank you for contributing @jakearchibald!

**Pull request merged**